CHANGELOG.cdf-tk.md

```diff
@@ -15,6 +15,12 @@ Changes are grouped as follows:
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## TBD
+
+### Fixed
+
+- Kind in Query files broke pull command. This is now fixed.
+
```

Comment:
Lets merge this after the new release process is in place. Move this to the description of the PR.